koji-setup/mash.sh

```diff
@@ -94,18 +95,18 @@ if [[ -e "$MASH_TRACKER_FILE" ]]; then
 else
 	MASH_BUILD_NUM=0
 fi
-KOJI_TAG="${KOJI_TAG:-"dist-clear"}"
 KOJI_REPO_PATH="$(realpath "$KOJI_DIR/repos/$KOJI_TAG-build/latest")"
 KOJI_BUILD_NUM="$(basename "$KOJI_REPO_PATH")"
 if [[ "$MASH_BUILD_NUM" -ne "$KOJI_BUILD_NUM" ]]; then
 	rm -rf "$MASH_DIR_NEW"
 	mkdir -p "$MASH_DIR_NEW"
 	create_dist_repos "$MASH_DIR_NEW"
-	if [[ -e "$MASH_TRACKER_DIR" ]]; then
-		mv "$MASH_TRACKER_DIR" "$MASH_DIR_OLD"
```

Comment:
Since `$MASH_DIR_OLD` needs to exist now as the move is moving it to a subdirectory rather than just renaming the directory to .old I think this needs a `mkdir -p $MASH_DIR_OLD`.

Reply:
ah yes! i will update that and $MASH_TRACKER_DIR! 
thanks!